anaconda-pkg-base/linux/defaults.Dockerfile

```diff
@@ -1,10 +1,8 @@
-ARG BASEVERSION=7
+FROM centos:7.9.2009@sha256:be65f488b7764ad3638f236b7b515b3678369a5124c47b8d32916d6487418ea4 AS base-amd64
 
-FROM centos:${BASEVERSION} AS base-amd64
+FROM amazonlinux:2.0.20240412.0@sha256:3bc2191fa2d5f467217e5ca947485bc203b73c02b9a23b164818fa6e37647625 AS base-arm64
 
-FROM amazonlinux:2 AS base-arm64
-
-FROM clefos:${BASEVERSION} AS base-s390x
+FROM clefos:7.7.1908 AS base-s390x
```

Comment:
```suggestion
FROM clefos:7.7.1908@sha256:611f82f69914a58ca4daaf3dd7905eef50bc10e146a501a4c5e49259fd87140a AS base-s390x
```

Reply:
its not that easy ... the other sha256s have been all looked up by renovate AND there are multiple different sha256 one can select from. renovate does not choose the layer sha256 but the manifest sha256, which references all the layers via sha256, i.e. "@sha256:611f82f699..." is a layer sha. The mainfest sha can be seen when selecting the tag https://hub.docker.com/layers/library/clefos/7.7.1908/images/sha256-611f82f69914a58ca4daaf3dd7905eef50bc10e146a501a4c5e49259fd87140a?context=explore ... so it is `sha256:611f82f69914a58ca4daaf3dd7905eef50bc10e146a501a4c5e49259fd87140a`.